clients/admin-ui/src/theme/global.scss

```
@@ -44,18 +44,21 @@ h6 {
 
 // Custom styles for dark submenus
 .ant-menu-dark .ant-menu-sub {
-  // The unselected submenu item should be light gray
-  > .ant-menu-item > .ant-menu-title-content a {
-    color: var(--fidesui-neutral-200);
+  > .ant-menu-item {
+    > .ant-menu-title-content a {
+      color: var(--fidesui-neutral-200);
+
+      &:focus-visible {
+        text-decoration: underline;
+      }
+    }
 
     &:hover,
     &:focus {
```

This `&:focus` will never apply because the `a` element is focusable but the menu item is not.
```suggestion
    &:focus-within {
```

---

Playing around with this, it still doesn't seem to work, probably because Ant's stuff is overriding. Might just want to remove this :focus altogether in favor of the underline

---

yup, I think it's because I'm overriding --ant-menu-dark-item-hover-bg which doesn't apply if it's not hovering. I will try setting the background color directly instead.

---

@gilluminate setting the background-color property worked for both hovering and focusing with keyboard.